microovn/api/certificates/regenerate_ca.go

```diff
@@ -29,7 +29,7 @@ func regenerateCaPut(s *state.State, r *http.Request) response.Response {
 	responseData := types.NewRegenerateCaResponse()
 
 	// Check that this is the initial node that received the request and recreate new CA certificate
-	if !client.IsForwardedRequest(r) {
+	if !client.IsNotification(r) {
```

Comment:
@gabrielmougard So I checked and the problem is not with `BATS`. Command `microovn certificates regenerate-ca` indeed does not work. It fails with
```
Error: command failed: failed to generate new CA: Put "http://control.socket/1.0/ca": context deadline exceeded
```
as the CI shows.

I also noticed that it absolutely obliterates the CPU :laughing:. So perhaps this change from `IsForwardedRequest` to `IsNotification` is not working as expected.

The way this command is supposed to work is that the original node that receives the request from client forwards the request to the rest of the nodes in the cluster. This condition is then used to distinguish whether the node received direct request from client (and should forward requests to all other cluster members), or it's the forwarded message and the node should just do its own thing.

The high CPU consumption suggests to me that even the servers that received the forwarded message then try to forward it again to everyone else, creating kind of a death spiral.

But that's just a guess. I didn't dive too deep into it.